backend/src/editor/editor.service.ts

```diff
@@ -28,35 +28,35 @@ export class EditorService {
   }
 
   async createDocument(roomId: string, template: string): Promise<void> {
-    const text = new Doc().getText();
+    const document = new Doc();
+    const text = document.getText(EDITOR_DOCUMENT_NAME);
     // Monaco Editor expects line endings to follow CRLF.
     text.insert(0, template.replace(/(?<!\r)\n/g, "\r\n"));
-    const delta = JSON.stringify(text.toDelta());
-    await this.redisService.setKey([EditorService.NAMESPACE], roomId, delta);
+    const update = Buffer.from(encodeStateAsUpdate(document)).toString(
+      "base64",
+    );
+    await this.redisService.setKey([EditorService.NAMESPACE], roomId, update);
```

Comment:
This part seems to be the same as below.
```suggestion
    await this.saveDocument(roomId, document);
```